src/Serializer/Normalizer/PhoneNumberNormalizer.php

```diff
@@ -82,11 +84,12 @@ public function supportsNormalization($data, $format = null)
      * {@inheritdoc}
      *
      * @throws UnexpectedValueException
+     * @return mixed
      */
     public function denormalize($data, $class, $format = null, array $context = [])
     {
         if (null === $data) {
-            return;
+            return null;
```

Comment:
Please remove this change as well `return;` was fine and it follows Sf coding style

Reply:
Hello done and pushed